arch/risc-v/src/common/crt0.c

```diff
@@ -88,6 +88,71 @@ static void sig_trampoline(void)
   );
 }
 
+/****************************************************************************
+ * Public Data
+ ****************************************************************************/
+
+#if defined(__cplusplus)
```

Comment:
Sorry for dummy question, but do we expect to compile this file with CXX?

Reply:
Yes, C++ applications use the same CRT

Reply:
But do we need to wrap `void _start(int argc, char *argv[])` with `extern "C"`? I can't find where `_start` is called.

Reply:
It is the process entry point, binfmt loads an .elf file and calls the entry point. Every elf file is compiled either with gcc or g++.

It does not matter if the name of start is name mangled, as it is not really "called", a jump to the address of _start is performed.

Reply:
I see. Thank you for explanation